.github/workflows/test.yaml

```diff
@@ -24,17 +24,22 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Install Packages
-      run: pip install -r requirements_dev.txt
+      run: |
+        pip install -r requirements.txt
+        pip install -r test-requirements.txt
 
     - name: Test
       env:
         BW_ACCOUNT_ID: ${{ secrets.BW_ACCOUNT_ID }}
         BW_USERNAME: ${{ secrets.BW_USERNAME }}
         BW_PASSWORD: ${{ secrets.BW_PASSWORD }}
+        BW_USERNAME_FORBIDDEN: ${{ secrets.BW_USERNAME_FORBIDDEN }}
+        BW_PASSWORD_FORBIDDEN: ${{ secrets.BW_PASSWORD_FORBIDDEN }}
         BW_VOICE_APPLICATION_ID: ${{ secrets.BW_VOICE_APPLICATION_ID }}
         BW_MESSAGING_APPLICATION_ID: ${{ secrets.BW_MESSAGING_APPLICATION_ID }}
         BW_NUMBER: ${{ secrets.BW_NUMBER }}
         USER_NUMBER: ${{ secrets.USER_NUMBER }}
+        VZW_NUMBER: ${{ secrets.VZW_NUMBER }}
```

Comment:
👍  assuming this is a Verizon Wireless Number!

Reply:
@bpateldx it is- but its my personal cell. We should change the value of this _but_ the team doesnt actually own any verizon wireless tns. TAC might - im going to ask them - but if i ever move my tn to a different provider this test would break

Reply:
Updated to be the BW owned TAC tns and added AT&T and T-Mobile numbers